modules/web-element.mjs

```diff
@@ -324,22 +324,188 @@ class PaddingLeft extends WebElementFunction {
  */
 class FontSize extends WebElementFunction{
     constructor(){
-        super("fontSize")
+        super("font-size")
     }
-    get(e){
-        var s = e.style.fontSize
-        return parseFloat(s)
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow;
+        var elem = doc.querySelector('#title')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('font-size')
+        return cssProp
     }
 }
+/**
+ * Function that extracts the font family
+ */
 class FontFamily extends WebElementFunction{
     constructor(){
-        super("fontFamily")
+        super("font-family")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow;
+        var elem = doc.querySelector('#title')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('font-family')
+        return cssProp
+    }
+}
+/**
+ * this is a simple exple to test h1's color is
+ */
+class CheckColorGreen extends WebElementFunction{   
+    constructor(){
+        super("color")
+    }
+    get(element){
+        //var element = document.querySelector('.heading')
+        var doc = element.ownerDocument; //Dom rérence
+        var win = doc.defaultView || doc.parentWindow; //window reference
+        var elem = doc.querySelector('.heading')
+        var cssPropVal = win.getComputedStyle(elem).getPropertyValue('color')
+        return cssPropVal;
+    }
+}
+
+
+/**
+ * Function that extracts the background-color of a DOM.
+ */
+class CheckBackgroundColor extends WebElementFunction {
```

Comment:
Should all functions be prepended with `Check` or not? 
This PR seems to add a lot of `Check` prefix, but I don't see the purpose/value of that change...

Reply:
It is not required to precede function names with check. just is used by me. so will removed it.